src/schema/validators/common.ts

```diff
@@ -567,7 +567,7 @@ export const MapProps = {
 export const CardBase = {
     border: {
         type: 'string',
-        enum: ['border', 'shadow', 'none'],
+        enum: ['line', 'shadow', 'none'],
```

Comment:
would old card with border `border` also work?

Reply:
It was always a property `line`. I think it's a mistake. You can check properties of CardBase